CHANGELOG.md

```diff
@@ -141,6 +160,10 @@ _Not yet tracked in this changelog._
 
 <!-- Links to pull requests and issues. Note that you can use "issues"
 in the URL for both issues and pull requests. -->
+[#203]: https://github.com/stm32-rs/stm32l0xx-hal/issues/203
+[#200]: https://github.com/stm32-rs/stm32l0xx-hal/issues/200
+[#197]: https://github.com/stm32-rs/stm32l0xx-hal/issues/197
+[#196]: https://github.com/stm32-rs/stm32l0xx-hal/issues/196
```

Comment:
All those links are broken.

```suggestion
[#203]: https://github.com/stm32-rs/stm32l0xx-hal/pull/203
[#200]: https://github.com/stm32-rs/stm32l0xx-hal/pull/200
[#197]: https://github.com/stm32-rs/stm32l0xx-hal/pull/197
[#196]: https://github.com/stm32-rs/stm32l0xx-hal/pull/196
```

Reply:
Ughh, that's new. In the past, `issue` would automatically redirect to `pull`.

Reply:
The links say `issues` though, not `issue`.

Reply:
Sorry, I meant to write `issues` 🙂

I'll update the PR now.

Reply:
Uh, this is weird. In my browser session, https://github.com/stm32-rs/stm32l0xx-hal/issues/203 fails. In an anonymous window (or with curl), it properly redirects to the PR.

Reply:
Oh, I was confused. I thought the correct link to an issue was `/issue/123`, but it's actually `/issues/123`. That's indeed something that used to work.

Reply:
> Uh, this is weird. In my browser session, #203 fails. In an anonymous window (or with curl), it properly redirects to the PR.

Can confirm, same for me! Probably a weird bug that only affects logged in users.